PHP/Language.md

```diff
@@ -7,34 +7,34 @@ Write data to the script’s output: echo vs print();
 
 Script termination. 
 
-Data types: 4 scalar types, 4 compound types, 2 special types. Pseudo-types. 
+Data types: 4 scalar types, 4 compound types, 2 special types. Pseudo-types. (*) 
 
-Type Casting (with casting operators and functions). Detecting Types. 
+Type Casting (with casting operators and functions). Detecting Types. (*)  
 
 Variables (naming, case-sensitivity). Variable variables.
 
-Constants. Possible ways to define. Possible values that they contain. Magic constants.
+Constants. Possible ways to define. Possible values that they contain. Magic constants. (*) 
 
 ### Operators
 Operator precedence.
 
-Arithmetic operators ($a**2 etc)
+Arithmetic operators ($a*2 etc)
```

Comment:
Do you really need replacing **Exponentiation** operator by **Multiplication**?

Reply:
Ha-ha, no :D. I just thought that it was a typo for a moment. I have never used it before. Will revert then, thanks!